src/components/TodoList.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC } from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { Todo } from '../types/Todo';
+import { TodoComponent } from './TodoComponent';
```

Comment:
Add this component to your commit. Currently it's absent on GitHub(
![image](https://user-images.githubusercontent.com/96409563/206900433-eb717f4b-28d4-4add-a307-3f97fb65db91.png)